### PR TITLE
Add wolfSSL sample

### PR DESCRIPTION
**Pull request by OYTIS**

### CODE REVIEW

**Comment by OYTIS** [comment]
**Hedwig** :owl: : starting a job.
                         ID `b4111993-4c4b-4a08-8f4e-5c4d1f68ebe5`

**Comment by OYTIS** [comment]
**Hedwig** said YES.
        Get log at https://a16d9c35.ngrok.io/jobs/b4111993-4c4b-4a08-8f4e-5c4d1f68ebe5/log
        Restart by posting `#hedwig restart`

File: cloud-support/custom/README.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ For the client to be able to find the engine, it should be run with `OPENSSL_CON
 
 Sample client using mbedtls. The client depends on the Trust Onboard library built with `MBEDTLS_SUPPORT` flag and dynamically-linked mbedtls library in default configuration.
 
+## client-wolfssl
+
+Sample client using wolfSSL. A few points need to be taken care of with respect to the wolfSSL library.
+
+  * Because of how wolfSSL is designed, siging algorithms not supported by Trust Onboard hardware should be disabled on the build stage.
```
Comment (rbeiter) [inline cloud-support/custom/README.md]: Small typo here - 'siging' -> 'signing'

**Comment by OYTIS** [comment]
I need to get the fixed URLs. Merge, as only documentation has changed.

**Pull request merged**